cmd/dhcpserver/list.go

```diff
@@ -49,7 +49,21 @@ var listCmd = &cobra.Command{
 		}
 
 		table := utils.NewTable()
-		table.Render(dhcpservers, nil)
+		table.SetHeader([]string{"ID", "Network ID", "Network Name", "Status"})
+		if len(dhcpservers) != 0 {
```

Comment:
can we have early return if length is 0 and move the content of this if block outside

Reply:
Addressed!